deployments/charts/kuma/templates/cp-deployment.yaml

```diff
@@ -42,34 +42,20 @@ spec:
               value: {{ .Release.Namespace | quote }}
             - name: KUMA_GENERAL_ADVERTISED_HOSTNAME
               value: {{ include "kuma.name" . }}-control-plane.{{ .Release.Namespace }}
-            {{- if ne .Values.controlPlane.mode "global" }}
-            - name: KUMA_SDS_SERVER_GRPC_PORT
-              value: "5677"
-            - name: KUMA_XDS_SERVER_GRPC_PORT
-              value: "5678"
-            - name: KUMA_BOOTSTRAP_SERVER_PORT
-              value: "5682"
-            - name: KUMA_SDS_SERVER_TLS_CERT_FILE
-              value: /var/run/secrets/kuma.io/kuma-sds/tls-cert/tls.crt
-            - name: KUMA_SDS_SERVER_TLS_KEY_FILE
-              value: /var/run/secrets/kuma.io/kuma-sds/tls-cert/tls.key
-            {{- end }}
+            - name: KUMA_GENERAL_TLS_CERT_FILE
+              value: /var/run/secrets/kuma.io/tls-cert/tls.crt
+            - name: KUMA_GENERAL_TLS_KEY_FILE
+              value: /var/run/secrets/kuma.io/tls-cert/tls.key
```

Comment:
OK where did all these ports go? Even if we merge them to one single port (is it the API?) then we need to make sure these are restricted on Global, as per the removed declaration here.

Reply:
No, the ports config here is gone because we can rely on defaults. There should not be defined here in the first place. Some old leftover.